packages/react-native-gesture-handler/src/v3/types.ts

```diff
@@ -1,17 +1,23 @@
 import { Animated, NativeSyntheticEvent } from 'react-native';
-import {
-  GestureEventPayload,
-  GestureTouchEvent,
-  HandlerStateChangeEventPayload,
-} from '../handlers/gestureHandlerCommon';
+import { GestureTouchEvent } from '../handlers/gestureHandlerCommon';
 import { HandlerCallbacks } from '../handlers/gestures/gesture';
+import { ValueOf } from '../typeUtils';
+import { State } from '../State';
 
-export type GestureUpdateEventWithData<T> = GestureEventPayload & {
+export interface NativeEventPayload {
+  handlerTag: number;
+  state: ValueOf<typeof State>;
```

Comment:
```suggestion
  state: State;
```

https://github.com/software-mansion/react-native-gesture-handler/blob/18be3b30617161f5d11d21a3b7e711571cb1fc39/packages/react-native-gesture-handler/src/State.ts#L13

https://github.com/software-mansion/react-native-gesture-handler/blob/18be3b30617161f5d11d21a3b7e711571cb1fc39/packages/react-native-gesture-handler/src/typeUtils.ts#L1

Reply:
Done, 0bfaa192dd09ac7a6c6fe923e4fc63a06ca6fe33